include/seqan/index/index_qgram_openaddressing.h

```diff
@@ -107,7 +107,7 @@ namespace seqan
         TShape            shape;        // underlying shape
         TCargo            cargo;        // user-defined cargo
         TBucketMap        bucketMap;    // bucketMap table (used by open-addressing index)
-        TSize            stepSize;    // store every <stepSize>'th q-gram in the index
+        TSize            stepSize{1};   // store every <stepSize>'th q-gram in the index
```

Comment:
why was this necessary? but alpha could be left as is?

Reply:
I could also do member init in the ctor,but is was easier to just initialise the member.

Reply:
But the member is initialized in the default c’tor or is it not?